src/steps/bulk-object-create.ts

```diff
@@ -3,9 +3,11 @@ import { Step, RunStepResponse, FieldDefinition, StepDefinition, RecordDefinitio
 
 export class BulkCreateObject extends BaseStep implements StepInterface {
 
-  protected stepName: string = 'Bulk create Salesforce Objects';
+  protected stepName: string = 'Bulk create Salesforce objects';
   protected stepExpression: string = 'bulk create salesforce (?<objName>[a-zA-Z0-9]+) objects';
   protected stepType: StepDefinition.Type = StepDefinition.Type.ACTION;
+  protected actionList: string[] = ['create'];
+  protected targetObject: string = 'Objects';

```

Comment:
Maybe this doesn't matter since we're planning to hide these from the UI, but I think there should be a clear distinction between this bulk step and the 'create object' step, more than just the word 'object' being plural. I think a user will be confused/annoyed if they click 'create', then click 'objects' and it expects a CSV. Maybe the target object could be 'many objects' or 'bulk objects'?